src/examples/main.ts

```diff
@@ -78,5 +80,6 @@ class App extends WidgetBase {
 
 const Projector = ProjectorMixin(App);
 const projector = new Projector();
+projector.__setProperties__({ registry });
```

Comment:
did we remove `.setProperties()` from the Projector mixin?  It seems dangerous to give an example of something that we say people should never use?

Reply:
ah yes we haven't removed it (can't because it extends `WidgetBase`) but we have added `setProperties` so that should change.

Reply:
my bad

Reply:
sorted